lib/stdio/handle.js

```diff
@@ -13,8 +13,8 @@ export const handleInput = (addProperties, options, isSync) => {
 	const stdioStreamsGroups = [[...stdinStreams, ...handleInputOptions(options)], ...otherStreamsGroups]
 		.map(stdioStreams => validateStreams(stdioStreams))
 		.map(stdioStreams => addStreamDirection(stdioStreams))
-		.map(stdioStreams => normalizeGenerators(stdioStreams))
 		.map(stdioStreams => handleStreamsEncoding(stdioStreams, options, isSync))
+		.map(stdioStreams => normalizeGenerators(stdioStreams))
```

Comment:
Moving this line allows re-using the normalization logic from `normalizeGenerators()`.